proposals/lip-0057.md

```diff
@@ -1469,6 +1469,20 @@ Here `x` is the BFT weight at which the elements `validatorList[0].bftWeight, ..
 
 More functions might be made available during implementation.
 
+#### getActiveValidatorsNumber
+
+```python
+def getActiveValidatorsNumber()-> uint32:
+    return NUMBER_ACTIVE_VALIDATORS
+```
+
+#### getRoundLength
+
+```python
+def getRoundLength()-> uint32:
+    return ROUND_LENGTH
+```
+
```

Comment:
Since we are adding these two, do you think we should add `getStandByValidatorsNumber()` also.

Reply:
Since we don't seem to need this for now, I would skip it.